assets/docker/mercury/nginx.conf

```diff
@@ -0,0 +1,24 @@
+events {}
+
+http {
+    upstream mercury-service-api {
+        server mercury-service-api:5000;
```

Comment:
Should this be 5001 to match the v0 service?

Reply:
I'm actually confused on why this is working. I could have sworn I had 5001 here. Investigating

Reply:
Experimented a bit, and this is working correctly. Changing it to 5001 results in a 502 bad gateway error.  I need to brush up on my docker networking, but I'm imagining something along the lines of `mercury-service-api:5000` mapping to the port internal to the network, rather than the external one.

Reply:
Mentioned in slack, but if anyone follows git blame to this PR and is curious, check out https://docs.docker.com/compose/networking/

Reply:
Thanks!